jdaviz/configs/imviz/plugins/rotate_canvas/rotate_canvas.py

```diff
@@ -50,6 +51,7 @@ def __init__(self, *args, **kwargs):
         self.hub.subscribe(self, RemoveDataMessage, handler=self._on_viewer_data_changed)
 
     @property
+    @deprecated(since="3.9", alternative="Links Control plugin")

```

Comment:
this annoyingly raises the deprecation error when just accessing `viz.plugins`, which is causing tests to fail.   Once #23 is merged, this could instead build on that infrastructure to delay the warning until when individual methods are called.

Reply:
#23 is in – might you have a chance to rebase and update?